roles/ci_gen_kustomize_values/templates/common/edpm-nodeset-values/values.yaml.j2

```diff
@@ -1,5 +1,22 @@
 ---
 # source: common/edpm-nodeset-values/values.yaml.j2
+{%- if cifmw_ci_gen_kustomize_values_primary_ip_version == 4 -%}
+{%- set _ipv = {'ipvX': 'ipv4',
+                'ip_vX': 'ip_v4',
+                'ansible_default_ipvX': 'ansible_default_ipv4',
+                'network_vX': 'network_v4',
+                'vX': 'v4',
+                'gw_vX': 'gw_v4',
+                'ipvX_ranges': 'ipv4_ranges'} -%}
+{%- else -%}
+{%- set _ipv = {'ipvX': 'ipv6',
+                'ip_vX': 'ip_v6',
+                'ansible_default_ipvX': 'ansible_default_ipv6',
+                'network_vX': 'network_v6',
+                'vX': 'v6',
+                'gw_vX': 'gw_v6',
+                'ipvX_ranges': 'ipv6_ranges'} -%}
+{%- endif -%}
```

Comment:
That sounds exactly like a thing we want to nudge in a "snippet", using the `include` jinja2 instruction, like there for instance:
https://github.com/openstack-k8s-operators/edpm-ansible/blob/f6f28a6d4f8b770624d6ea33f4f67bb8c4bb2fb3/roles/edpm_nftables/templates/ruleset.j2#L13

We can get something like `template/snippets/ip-version.j2` and include it?

Reply:
indeed, I had forgotten about include in jinja2. I'll iron out some other issues and change to use an inlcude.

Thanks for the suggestion!

Reply:
ok, so include did not work. But jinja2 also support imports as well, so allows us o get the one file with the mappings.

Reply:
Marking as resolved - something went sideways with the import / include approach.
I moved the map building to the vars instead.